groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/lanthanides/RareEarthProcessing.groovy

```diff
@@ -0,0 +1,1046 @@
+        .fluidInputs(fluid('tributyl_phosphate') * 500)
+        .fluidOutputs(fluid('mona_cerium_extract') * 500)
+        .fluidOutputs(fluid('mona_cefree_re_nitrates_solution') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+
+BLENDER.recipeBuilder()
+        .fluidInputs(fluid('mona_cefree_re_nitrates_solution') * 1000)
+        .fluidInputs(fluid('rare_earth_extraction_mixture') * 100)
+        .fluidOutputs(fluid('mona_lree_concentrate') * 50)
+        .fluidOutputs(fluid('mona_dehpa_extract') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+
+MIXER.recipeBuilder()
+        .fluidInputs(fluid('hydrochloric_acid') * 50)
+        .fluidInputs(fluid('mona_dehpa_extract') * 1000)
+        .fluidOutputs(fluid('rare_earth_extraction_mixture') * 100)
+        .fluidOutputs(fluid('mona_re_chlorides_solution') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+
+BLENDER.recipeBuilder()
+        .fluidInputs(fluid('mona_re_chlorides_solution') * 1000)
+        .fluidInputs(fluid('rare_earth_separation_mixture') * 100)
+        .fluidOutputs(fluid('mona_mree_concentrate') * 500)
+        .fluidOutputs(fluid('mona_ehehpa_extract') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+
+MIXER.recipeBuilder()
+        .fluidInputs(fluid('hydrochloric_acid') * 50)
+        .fluidInputs(fluid('mona_ehehpa_extract') * 1000)
+        .fluidOutputs(fluid('rare_earth_separation_mixture') * 100)
+        .fluidOutputs(fluid('mona_hree_concentrate') * 40)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+// Xenotime Concentrate Processing
+
+MIXER.recipeBuilder()
+        .inputs(ore('dustPotassiumPermanganate') * 1)
+        .fluidInputs(fluid('nitric_acid') * 1150)
+        .fluidInputs(fluid('xeno_re_nitrates_solution') * 1000)
+        .fluidOutputs(fluid('xeno_ox_re_nitrates_solution') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+
+BLENDER.recipeBuilder()
+        .fluidInputs(fluid('xeno_ox_re_nitrates_solution') * 1000)
+        .fluidInputs(fluid('tributyl_phosphate') * 100)
+        .fluidOutputs(fluid('xeno_cerium_extract') * 100)
+        .fluidOutputs(fluid('xeno_cefree_re_nitrates_solution') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+
+BLENDER.recipeBuilder()
+        .fluidInputs(fluid('xeno_cefree_re_nitrates_solution') * 1000)
+        .fluidInputs(fluid('rare_earth_extraction_mixture') * 1000)
+        .fluidOutputs(fluid('xeno_lree_concentrate') * 20)
+        .fluidOutputs(fluid('xeno_dehpa_extract') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+
+MIXER.recipeBuilder()
+        .fluidInputs(fluid('hydrochloric_acid') * 50)
+        .fluidInputs(fluid('xeno_dehpa_extract') * 1000)
+        .fluidOutputs(fluid('rare_earth_extraction_mixture') * 1000)
```

Comment:
What particularly makes this step require a full bucket of rare earth separation mixture as opposed to the 100L used elsewhere?